src/Stratis.FederatedSidechains.AdminDashboard/Program.cs

```diff
@@ -12,7 +12,7 @@ public static void Main(string[] args)
 
         public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
-                .UseUrls("http://localhost:1989")   //TODO: temporary port
+                .UseUrls("http://localhost:37000")
```

Comment:
After this commit you could potentially create a feature that will start this on the fed node startup.

Reply:
Why not, do you know the location or something that can help me ?